core/src/test/java/io/confluent/kafka/schemaregistry/rest/RestApiTest.java

```diff
@@ -395,6 +394,29 @@ public void testGetVersion() throws Exception {
                  restApp.restClient.getLatestVersion(subject).getSchema());
   }
 
+  @Test
+  public void testGetLatestVersionSchemaOnly() throws Exception {
+    List<String> schemas = TestUtils.getRandomCanonicalAvroString(2);
+    String subject = "test";
+    TestUtils.registerAndVerifySchema(restApp.restClient, schemas.get(0), 1, subject);
+    TestUtils.registerAndVerifySchema(restApp.restClient, schemas.get(1), 2, subject);
+
+    assertEquals("Latest schema should be the same as version 2",
+                 schemas.get(1),
+                 restApp.restClient.getLatestVersionSchemaOnly(subject));
+  }
+
+  @Test
+  public void testGetVersionSchemaOnly() throws Exception {
+    List<String> schemas = TestUtils.getRandomCanonicalAvroString(1);
+    String subject = "test";
+    TestUtils.registerAndVerifySchema(restApp.restClient, schemas.get(0), 1, subject);
+
+    assertEquals("Latest schema should be the same as version 1",
```

Comment:
The message here seems to be incorrect as the test seems to be oriented towards retrieving a specific version of the schema rather than the latest version.